lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<EOM
+
+'#{METADATA_REPOSITORY}' repository not found.
+Try running the command:
+  git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git ../#{METADATA_REPOSITORY}
```

Comment:
There are multiple ways that this could work, but I like that this one gives an immediate resolution to the current problem. Just copy/paste that line and you're good to go again.

Good spot on the relative path issue, I'll replace that with an absolute path.